cloudmapper.py

```diff
@@ -56,6 +56,34 @@ def run_gathering(arguments):
     gather(args)
 
 
+def run_configure(arguments):
+    from cloudmapper.configure import configure
+    if len(arguments) == 0:
+        exit("ERROR: Missing action for configure. Should be in {add-cidr|add-account|remove-cidr|remove-account}")
+        return
+    action = arguments[0]
+    arguments = arguments[1:]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config-file", help="Path to the config file",
+                        required=True, type=str)
```

Comment:
Let's not set this as `required=True` and instead set `default="config.json"` as done in https://github.com/duo-labs/cloudmapper/blob/d74329044f73e7c5c03446be2da1fc34d539c49d/cloudmapper.py#L65

Reply:
Nice catch ! 👍